src/query/settings/src/settings_default.rs

```diff
@@ -300,18 +300,30 @@ impl DefaultSettings {
                     possible_values: None,
                     mode: SettingMode::Both,
                 }),
-                ("spilling_bytes_threshold_per_proc", DefaultSettingValue {
```

Comment:
break change?

Reply:
Just change the name. To distinguish aggregate spill and sort spill.